neurom/core/__init__.py

```diff
@@ -28,9 +28,8 @@
 
 """Core functionality and data types of NeuroM."""
 
-from .tree import Tree
 from .types import NeuriteType, NeuriteIter
-from ._soma import Soma, make_soma, SomaError
+from ._soma import Soma
```

Comment:
why using relative imports ?

Reply:
because NeuroM v1. I didn't clean the code in general as I don't like to mix multiple concerns in the single PR.